lib/java.gradle

```diff
@@ -187,6 +187,9 @@ configure(projectsWithFlags('java')) {
                                      "targetJavaVersion(${project.ext.targetJavaVersion})")
             testTask.enabled = false
         }
+        if (testTask.enabled) {
+            project.ext.configureFlakyTests(testTask)
```

Comment:
Please note that I didn't remove the configureFlakyTests closure because it's a breaking.

Reply:
To be honest, I think 1) nobody is using this method 2) migration is easy because users can just remove it and use `configureCommonTestSettings`.
I guess it's fine to leave though and remove later.